packages/hdwallet-portis/src/portis.ts

```diff
@@ -181,7 +181,7 @@ export class PortisHDWallet implements HDWallet, ETHWallet, BTCWallet {
     return this.info.describePath(msg)
   }
 
-  public async getPublicKeys(msg: GetPublicKey[]): Promise<PublicKey[]> {
```

Comment:
Why change portis?

Reply:
The function definition in the HDWallet interface changed.

Reply:
less risk to just leave it

Reply:
Should behave exactly the same way.

Reply:
It changed adding `| null`

no need to touch portis please

Reply:
Right, it matches HDWallet definition in core.

Reply:
its less risk to not touch this line

Reply:
@elmutt it'll be fine, no risk. Types in typescript are just an annotation that the compiler checks, so if it builds, it's good.